cli.js

```diff
@@ -27,19 +36,34 @@ const cli = meow(`
 
 	Options
 	  --copy -c  Copy the first emoji to the clipboard
+	  --skin-tone -s  Set the default skin tone of the emoji (0 to 5)
 
 	Run it without arguments to enter the live search
+	Use Up/Down keys during live search to change the skin tones
 `, {
 	boolean: [
 		'copy'
 	],
 	alias: {
-		c: 'copy'
+		c: 'copy',
+		s: 'skinTone'
 	}
 });
 
+if (typeof cli.flags.skinTone === 'number') {
```

Comment:
We could coerce the value to the initial object itself. Additionally, I don't think adding validation is necessary.

```js
if (cli.flags.skinTone != null) {
  cli.flags.skinTone = Math.max(0, Math.min(5, Number(cli.flags.skinTone));
}
```